apps/els_lsp/src/els_unused_includes_diagnostics.erl

```diff
@@ -64,7 +64,17 @@ find_unused_includes(#{uri := Uri} = Document) ->
     , type_application
     , export_type_entry
     ]),
-  IncludedUris = els_diagnostics_utils:included_uris(Document),
+  IncludedUris0 = els_diagnostics_utils:included_uris(Document),
+  ExcludeUnusedIncludes =
```

Comment:
Maybe do this file path expansion in the `els_config` module and print a warning there if the header file is not found?

Reply:
I would imagine developers reusing their list of `exclude_unused_includes` across multiple projects (which may not include all entries), so that should not be necessary.